ci/build-freebsd.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -e
+
+export CFLAGS="$CFLAGS -isystem/usr/local/include"
```

Comment:
Otherwise looks good, but just verifying that this is not a mistyping of `-I` (capital _i_). Also why is /usr/local being added, just out of curiosity?

Reply:
> verifying that this is not a mistyping of `-I` (capital _i_).

`-isystem` is used to
- Put the directory at the end of include search list
- Ignore warnings in headers outside of the project

https://gcc.gnu.org/onlinedocs/cpp/System-Headers.html

> why is /usr/local being added, just out of curiosity?

Base compiler doesn't look where package manager installs headers by default.
https://wiki.freebsd.org/WarnerLosh/UsrLocal#Include_paths

 For example, mpv looks for `gbm.h` without consulting `pkg-config gbm --cflags`:
```
$ ./waf configure --enable-egl-drm
Checking for gbm.h                                                        : no
Checking for GBM                                                          : gbm.h not found
Checking for OpenGL DRM EGL Backend                                       : gbm not found
You manually enabled the feature 'egl-drm', but the autodetection check failed.
```

Reply:
OK, thanks for confirmation.

So package manager goes into `/usr/local`, funky.